src/components/accordion.js

```diff
@@ -20,17 +20,19 @@ class Accordion extends Component {
         <a tabIndex='0' onClick={this.toggle}>
           {this.state.collapse ? (
             <small className='text-dark'>
-              <FontAwesomeIcon icon={faCaretDown} className='mr-1' />
+              <FontAwesomeIcon icon={faCaretDown} className='mr-2' />
               <b>{this.props.title}</b>
             </small>
           ) : (
             <small className='text-muted'>
-              <FontAwesomeIcon icon={faCaretRight} className='mr-1' />
+              <FontAwesomeIcon icon={faCaretRight} className='mr-2' />
               {this.props.title}
             </small>
           )}
         </a>
-        <Collapse isOpen={this.state.collapse}>{this.props.children}</Collapse>
+        <Collapse className='pl-3 py-2' isOpen={this.state.collapse}>
```

Comment:
is this padding here? whatever you did drastically improved the spacing within the expandable bullet points so it doesn't look as crammed anymore. Thank you